blackbox/handlers/storage/s3.py

```diff
@@ -13,6 +13,8 @@
 
 
 class S3(BlackboxStorage):
+    """Maintain S3 storage API."""
```

Comment:
Classes don't follow quite the same conventions as methods, and we usually don't use imperative mood when we write these. Instead, we provide a high-level _summary_ of what the class is used for.

```suggestion
    """A storage handler for S3-compatible APIs."""
```